jwt-bcp/draft-ietf-oauth-jwt-bcp.md

```diff
@@ -326,6 +337,8 @@ ECDH-ES ephemeral public key (epk) inputs should be validated according to the r
 chosen elliptic curve. For the NIST prime-order curves P-256, P-384 and P-521, validation MUST
 be performed according to Section 5.6.2.3.4 "ECC Partial Public-Key Validation Routine" of
 NIST Special Publication 800-56A revision 3 [nist-sp-800-56a-r3].
+Likewise, if the "X25519" or "X448" {{RFC8037}} algorithms are used,
+then the values MUST be validated according {{RFC7748}}.
```

Comment:
RFC 8037 does not reference RFC 7748 for validation of encrypted/signed values, and for good reason. It actually explains why we don't care about sender-manipulated public keys.

We should however refer the reader to the security considerations of both of these RFCs.